Default values are of the wrong type

**What steps did you take and what happened:**
[A clear and concise description of what the bug is, and what commands you ran.]
Two values in the values.yaml `initContainers` and `additionalVolumes` are defaulted to an empty array. They should be defaulted to an empty dictionary instead.

```
coalesce.go:223: warning: destination for hedera-mirror-common.zfs.zfsNode.initContainers is a table. Ignoring non-table value ([])
coalesce.go:223: warning: destination for hedera-mirror-common.zfs.zfsNode.additionalVolumes is a table. Ignoring non-table value ([])
coalesce.go:223: warning: destination for hedera-mirror-common.zfs.zfsNode.initContainers is a table. Ignoring non-table value ([])
coalesce.go:223: warning: destination for hedera-mirror-common.zfs.zfsNode.additionalVolumes is a table. Ignoring non-table value ([])
coalesce.go:223: warning: destination for hedera-mirror-common.zfs.zfsNode.initContainers is a table. Ignoring non-table value ([])
coalesce.go:223: warning: destination for hedera-mirror-common.zfs.zfsNode.additionalVolumes is a table. Ignoring non-table value ([])
```

**What did you expect to happen:**

Not receive warnings when deploying helm chart.

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other [Pastebin](https://pastebin.com/) is fine.)

* `kubectl logs -f openebs-zfs-controller-0 -n kube-system -c openebs-zfs-plugin`
* `kubectl logs -f openebs-zfs-node-[xxxx] -n kube-system -c openebs-zfs-plugin`
* `kubectl get pods -n kube-system`
* `kubectl get zv -A -o yaml`

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**
- ZFS-LocalPV version
- Kubernetes version (use `kubectl version`):
- Kubernetes installer & version:
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`):

--- a/changelogs/unreleased/505-default-chart-values.md
+++ b/changelogs/unreleased/505-default-chart-values.md
@@ -1,0 +1,1 @@
+correct the default values for initContainers and additionalVolumes

--- a/deploy/helm/charts/Chart.yaml
+++ b/deploy/helm/charts/Chart.yaml
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: zfs-localpv
 description: Helm chart for CSI Driver for dynamic provisioning of ZFS Persistent Local Volumes. For instructions on how to use this helm chart, see - https://openebs.github.io/zfs-localpv/
-version: 2.4.1
-appVersion: 2.4.1
+version: 2.4.2
+appVersion: 2.4.2
 icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/icon/color/openebs-icon-color.png
 home: https://openebs.io/
 keywords:

--- a/deploy/helm/charts/values.yaml
+++ b/deploy/helm/charts/values.yaml
@@ -60,15 +60,15 @@ zfsNode:
   # For example:
   # allowedTopologyKeys: "kubernetes.io/hostname,openebs.io/rack"
   allowedTopologyKeys: "All"
-  initContainers: []
-  additionalVolumes: []
+  initContainers: {}
+  additionalVolumes: {}
 
 # zfsController contains the configurables for
 # the zfs controller statefulset
 zfsController:
   componentName: openebs-zfs-controller
-  initContainers: []
-  additionalVolumes: []
+  initContainers: {}
+  additionalVolumes: {}
   replicas: 1
   serviceName: openebs-zfs
   resizer:

